solidity_contracts/src/PlainOpcodes/Counter.sol

```diff
@@ -17,17 +17,21 @@ contract Counter {
         count += 1;
     }
 
-    function decUnchecked() public {
-        unchecked {
-            count -= 1;
-        }
+    function dec() public greaterThanZero {
+        count -= 1;
     }
 
     function decInPlace() public greaterThanZero {
         count--;
     }
 
-    function dec() public greaterThanZero {
+    function decUnchecked() public {
+        unchecked {
+            count -= 1;
+        }
+    }
+
+    function decUnsafe() public {
```

Comment:
what do we mean by unsafe? that it wraps around?
i never really looked into, i just remember SafeMath was integrated into Solidity 0.8 or smth and now integers are just "safe"